doc/USING.rst

```diff
@@ -12,7 +12,7 @@ Inference using a DeepSpeech pre-trained model can be done with a client/languag
 Running ``deepspeech`` might, see below, require some runtime dependencies to be already installed on your system:
 
 
-* ``sox`` - The Python and Node.JS clients use SoX to resample files to 16kHz.
+* ``sox`` - The Python and Node.JS clients use SoX to resample files to 16kHz. On MacOS, the Command-Line client will also require SoX to be installed.
```

Comment:
This is wrong, you need `libsox` for the C++ client, not `sox`.

Reply:
This distinction is meaningless on macOS, `libsox` == `brew install sox`.